HLTrigger/Configuration/python/deepTauAtHLT.py

```diff
@@ -0,0 +1,193 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
What is the purpose of this new file, and why does it need to be added under HLTrigger/Configuration, and not RecoTauTag/RecoTau or RecoTauTag/HLTProducers?

Reply:
True there is no need to put it in HLTrigger/Configuration. The included helper function was used to test the HLT setup and serves as sample configuration at least until deepTau is deployed in the HLT paths via confDB. Let me move it to RecoTauTag/HLTProducers